src/nkr_proxy/proxy_app.py

```diff
@@ -198,15 +215,33 @@ def search_index(user_restriction_level, entitlements, search_query):
 
     for n_retry, index_host in enumerate(settings.INDEX_HOSTS):
 
-        full_index_url = '%s/%s/%s' % (index_host, settings.INDEX_NAME, search_query)
+        #full_index_url = '%s/%s/%s' % (index_host, settings.INDEX_NAME, search_query)
 
-        logger.debug(full_index_url)
+        #logger.debug(full_index_url)
 
         try:
-            response = http_request(
-                full_index_url,
-                auth=(settings.INDEX_USERNAME, settings.INDEX_PASSWORD)
-            )
+            if method == 'get':
+                full_index_url = '%s/%s/%s' % (index_host, settings.INDEX_NAME, search_query)
+                response = http_request(
+                    full_index_url,
+                    method=method,
+                    auth=(settings.INDEX_USERNAME, settings.INDEX_PASSWORD)
+                )
+                logger.debug(full_index_url)
+            if method == 'post':
+                full_index_url = '%s/%s/select' % (index_host, settings.INDEX_NAME)
+                # take substring of search_query in order to ignore select? in the beginning of the query
```

Comment:
Just clarification: is 'select?' removed or just 'select' ie. will '?' be left in the beginning of query?